Dockerfile

```diff
@@ -16,6 +16,24 @@
 # or DOCKER_BUILDKIT=1 docker build -o /tmp/out . --target=buster
 # Generated tarball(s) will end up in /tmp/out
 
+FROM debian:bullseye AS bullseye-build
+
+# TODO: Factor out the common code without rerunning apt-get on every build.
```

Comment:
Minor, if we're keeping this TODO around it's probably fine to just leave it on the first entry and remove the duplicated one from the buster section.

Reply:
Done.